tls/test_tls_cert.py

```diff
@@ -759,6 +759,162 @@ def test(self):
                 self.assertTrue(x509_check_cn(hs.hs.server_cert[0], expected_cert))
 
 
+class WrkTestsMultipleVhosts(tester.TempestaTest):
+    NGINX_CONFIG = """
+pid ${pid};
+worker_processes  auto;
+
+events {
+    worker_connections   1024;
+    use epoll;
+}
+
+http {
+    keepalive_timeout ${server_keepalive_timeout};
+    keepalive_requests ${server_keepalive_requests};
+    sendfile         on;
+    tcp_nopush       on;
+    tcp_nodelay      on;
+
+    open_file_cache max=1000;
+    open_file_cache_valid 30s;
+    open_file_cache_min_uses 2;
+    open_file_cache_errors off;
+
+    # [ debug | info | notice | warn | error | crit | alert | emerg ]
+    # Fully disable log errors.
+    error_log /dev/null emerg;
+
+    # Disable access log altogether.
+    access_log off;
+
+    server {
+        listen        ${server_ip}:8000;
+
+        location / {
+            return 200;
+        }
+        location /nginx_status {
+            stub_status on;
+        }
+    }
+}
+"""
+
+    clients = [
+        {
+            "id": "wrk",
+            "type": "wrk",
+            "ssl": True,
+            "addr": f'{tf_cfg.cfg.get("Tempesta", "ip")}:443',
+        },
+    ]
+
+    backends = [
+        {
+            "id": "nginx",
+            "type": "nginx",
+            "port": "8000",
+            "status_uri": "http://${server_ip}:8000/nginx_status",
+            "config": NGINX_CONFIG,
+        }
+    ]
+
+    def config_changer(self):
+        self.set_second_config()
+        self.set_first_config()
+        self.set_second_config()
+        self.set_first_config()
+        self.set_second_config()
```

Comment:
These configs are identical. Is this expected? Maybe just use `self.get_tempesta().reload()`?

Reply:
Refurbish config_changer function